_stbt/core.py

```diff
@@ -2160,6 +2163,26 @@ def _tesseract_version(output=None):
 
 def _tesseract(frame, region, mode, lang, _config,
                user_patterns=None, user_words=None):
+    """Cache the OCR result of the most recent frame so that you can call
+    `match_text` multiple times on the same frame, cheaply.
+    """
+    global _memoise_tesseract_output
+    h = unicode(hash(
+    (0, '_tesseract', _memoise_tesseract_version,
+     _image_hash(frame), region, mode, lang,
```

Comment:
Good point. Specifically if you have lossless image capture and you're doing `wait_until(lambda: match_text(..., region=...))` where `region` is a UI element that rarely changes, and outside the region you have live TV.